server/tests/userRoute.spec.js

```diff
@@ -15,7 +12,7 @@ describe("GET Users", () => {
 });
 
 describe("POST new user", () => {
-  it("Adds a new user to the User table", async () => {
+  it("Adds a new user to the User table", async done => {
```

Comment:
Nice use of `done`!

Reply:
I was hoping `done` was going to solve that issue, but no dice. I also added extra flags in the testing suite, but no luck there either.